09-student_guide.Rmd

```diff
@@ -0,0 +1,277 @@
+
+::: {.dictionary}
+**Breakout Box: Learn more about phylogenetically informative sites**
+
+You may be wondering why the total branch length changes (31) is different from the number of site patterns listed in the phyDat summary (46). This is because only some of the 46 site patterns are what we call _phylogenetically informative_.
+
+Let's look at a short example:
+
+cat     GC*A* TT*C*
+
+dog     GC*C* TT*C*
+
+frog    GC*C* TT*A*
+
+snake   GC*C* TT*A*
+
+In this collection of sequences, there are 2 site patterns (both in bold), but only one is phylogenetically informative because it tells us about the relatedness between the species. While the cat sequence is different from the others at the third site, this doesn't actually give us information about the relationships among the four species. The sixth site, however, tells us that the cat and dog share a mutation that frog and snake do not. We can interpret that to mean the cat and dog are more closely related to each other than either is to the frog or the snake. Thus, this particular site pattern gives us information we can use to build the phylogenetic tree.
+:::
+
+::: {.reflection}
+QUESTIONS:
+
+7. what is the branch length distance between the beta variant and the alpha variant?
+
+8. Is the delta or the beta variant more closely related to the Wuhan reference sequence? How do you know?
+
+9. What is the longest branch length in the tree? What does this mean for the amount of variability seen in that variants versus the others?
+:::
+
+Now let's load and view a tree containing all nine SARS-CoV-2 samples.
+
+```{r, message = FALSE, warning = FALSE}
+spike.tree_full <- read.tree("sars_tree_full.tre")
+
+plot(spike.tree_full)
+edgelabels(spike.tree_full$edge.length)
+```
+
+What do these branch lengths tell us about the relationship among the SARS-CoV-2 variants?
+
+::: {.reflection}
+QUESTIONS:
+
+10. Did adding the omicron samples change the branch length distances between the original five samples? What is the branch length distance between the delta and beta variants now?
+
+11. What is the branch length distance between the omicron clade and its closest relative?
+
+12. The covid vaccines were originally designed based on the Wuhan reference sequence of the spike protein. Using what you have learned about the phylogenetic tree of SARS-CoV-2 variants, can you explain why these initial vaccines were less effective at protecting against the omicron variants than they were against the delta variant?
+:::
+
+## Exercise Four: Do other protein-coding regions show us the same phylogenetic relationships?
+
+The SARS-CoV-2 genome contains 6 protein-coding regions. So far, we've been working with genetic data from the region that codes for spike protein. (The spike protein is the part of the virus that sticks out to form the characteristic spikes on the outside of the SARS-CoV-2 virus.) However, you could also use any of the other 5 protein-coding regions for this exercise. 
```

Comment:
Could be cool to have a picture of the virus and point to the spike protein when it's first mentioned above

Reply:
YES